data/net/minecraft/world/entity/Entity.mapping

```diff
@@ -139,13 +149,21 @@ CLASS net/minecraft/world/entity/Entity
 		ARG 1 livingEntity
 	METHOD getEncodeId ()Ljava/lang/String;
 		COMMENT Returns the string that identifies this Entity's class
+	METHOD getExitPortal (Lnet/minecraft/server/level/ServerLevel;Lnet/minecraft/core/BlockPos;Z)Ljava/util/Optional;
+		ARG 1 destination
+		ARG 2 findFrom
```

Comment:
I'm pondering whether this should be renamed slightly to relay the fact that this is the center position from which the exit portal finding starts.

Reply:
I would use a javadoc in that case to relay the fact.

Reply:
I have added javadoc for it

Reply:
Please use a javadoc for the parameter instead of putting the information in the method javadoc.

Reply:
I moved javadoc to parameter